backend/battles/helpers.py

```diff
@@ -30,6 +30,15 @@ def get_battle_result(battle):
     creator_victory_points = 0
     opponent_victory_points = 0
 
+    # check if opponent has pokemons
```

Comment:
Any reason why this is done for the opponent and not for the creator?

Reply:
Yes, to fill the property winner in a Battle object, this function is executed. As I'm using `winner is None` as a check in the template, if I don't have this check for a battle which has no opponent's pokemons, I receive an error because it tries to run the battle.